packages/fxa-payments-server/src/components/fields.tsx

```diff
@@ -75,6 +77,18 @@ export const Field = ({
   );
   /* eslint-enable react-hooks/exhaustive-deps */
 
+  const focusElement =
```

Comment:
Gave this a try, thinking that maybe re-running the effect when the available element ref changes would help. Doesn't seem to have done, though :/